### DISCUSSION TITLE
Enable issues creation for repository members only

### DISCUSSION THREAD

**Discussion by antonkomarev** [community/community#29674] [Projects and Issues]
I want to make community to use Discussions, but keep issues creation for internal members.

It might be useful to allow to create issues for repository members only. To let community see issues, comment in them, but not create them.

**Comment by GabyCT** [comment]
According to this https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-an-issues-only-repository
```
Creating an issues-only repository
GitHub does not provide issues-only access permissions, but you can accomplish this using a second repository which contains only the issues.
```

**Reply by antonkomarev** [reply]
Thanks. That's exactly what I'm doing: separate repository just for issues/discussions. It's good when you have to completely hide issues from community. But all I want — restrict non-members to create issues. But they still be able to read them and participate in comments.
